server/handlers.go

```diff
@@ -500,15 +500,15 @@ func (s *Server) putHandler(w http.ResponseWriter, r *http.Request) {
 	relativeURL, _ := url.Parse(path.Join(s.proxyPath, token, filename))
 	deleteURL, _ := url.Parse(path.Join(s.proxyPath, token, filename, metadata.DeletionToken))
 
-	w.Header().Set("X-Url-Delete", resolveURL(r, deleteURL))
+	w.Header().Set("X-Url-Delete", resolveURL(r, deleteURL, s.proxyPort))
 
-	fmt.Fprint(w, resolveURL(r, relativeURL))
+	fmt.Fprintln(w, resolveURL(r, relativeURL, s.proxyPort))
```

Comment:
please, keep `fmt.Fprint`
this is a long establihed behaviour of transfer.sh that returned url is not newline trailed

Reply:
Sure I can revert it. Interesting to note that it already is Fprintln for the POST handler, but not the PUT handler which seems a bit conflicting; with that said this PR obviously isn't trying to address that, so I can change it back